ff4j-web/pom.xml

```diff
@@ -99,6 +99,22 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-test</artifactId>
```

Comment:
`spring-test` is added here just to be able `MockPageContext` in unit test (but spring is not used in production code).